docs/S3.md

```diff
@@ -6,6 +6,8 @@ If using the default endpoint, you **must** configure the region using the `SCCA
 
 If your endpoint requires HTTPS/TLS, set `SCCACHE_S3_USE_SSL=true`. If you don't need a secure network layer, HTTP (`SCCACHE_S3_USE_SSL=false`) might be better for performance.
 
+Enable server side encryption with `SCCACHE_S3_SERVER_SIDE_ENCRYPTION=true`.
```

Comment:
Would it be possible to add a link to the Doc to explain what is this encryption? Thanks

Reply:
Maybe this docs will be helpful: https://opendal.apache.org/docs/services/s3#server-side-encryption